client/settings/payment-method-icon/style.scss

```diff
@@ -4,16 +4,21 @@
 	vertical-align: middle;
 	max-width: 160px;
 
-	svg {
-		width: 41px;
-		height: auto;
-	}
-
 	@include breakpoint( '>660px' ) {
 		max-width: inherit;
 	}
 
 	&__label {
 		margin-left: 4px;
 	}
+
+	svg {
+		width: 38px;
+		height: 24px;
+	}
+}
+.woocommerce-payments__payment-method-has-icon-border {
+	box-shadow: 0 0 0 1px #ddd;
+	width: 38px;
+	height: 24px;
```

Comment:
(minor) so that it looks less squared, especially around Sofort or Sepa

Before:
![Screen Shot 2021-07-26 at 3 24 31 PM](https://user-images.githubusercontent.com/273592/127054130-0a10910b-5c31-4a5c-a403-16193f8eab3d.png)


After:
![Screen Shot 2021-07-26 at 3 24 14 PM](https://user-images.githubusercontent.com/273592/127054096-602eea30-9ef5-4dcc-add6-b0fdab58f587.png)

```suggestion
	box-shadow: 0 0 0 1px #ddd;
	border-radius: 2px;
	width: 38px;
	height: auto;
```

Reply:
Added 2px radius